templates/_navbar_wagon.html.erb

```diff
@@ -1,35 +1,36 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
-  <%= link_to "#", class: "navbar-brand" do %>
-    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-  <% end %>
+  <div class="container">
```

Comment:
Shouldn't put `container-fluid` here instead of `container` to have a full width navbar? :thinking: 

`container` screenshot :point_down: 

![image](https://user-images.githubusercontent.com/37899667/138293716-51f5069c-dcbc-4523-ac6b-8344b7bb3c8f.png)


`container-fluid` screenshot :point_down: 
![image](https://user-images.githubusercontent.com/37899667/138293681-86b8ccce-c0e1-4a95-82bd-9b99e1e894db.png)